src/plugins/minimal_scene/MinimalScene.cc

```diff
@@ -328,6 +339,38 @@ void IgnRenderer::Render(RenderSync *_renderSync)
   // update and render to texture
   this->dataPtr->camera->Update();
 
+  if (!this->cameraViewController.empty())
+  {
+    std::string viewControlService = "/gui/camera/view_control";
+    transport::Node node;
+    std::function<void(const msgs::Boolean &, const bool)> cb =
+        [&](const msgs::Boolean &/*_rep*/, const bool _result)
+    {
+      if (!_result)
+        ignerr << "Error setting view controller" << std::endl;
+      else
+      {
+        this->cameraViewController = "";
+      }
+    };
+
+    msgs::StringMsg req;
+    if (this->cameraViewController.find("orbit") != std::string::npos)
+    {
+      req.set_data("orbit");
+    }
+    else if (this->cameraViewController.find("ortho") != std::string::npos)
+    {
+      req.set_data("ortho");
+    }
+    else
+    {
+      ignerr << "Unknown view controller selected: "
+             << this->cameraViewController << std::endl;
+    }
```

Comment:
Do we need this error checking if it's already done for us here? https://github.com/gazebosim/gz-gui/blob/0e4a198062f814573b32164804279237d93e2d6d/src/plugins/interactive_view_control/InteractiveViewControl.cc#L337-L341

It will be easier to maintain if we support more views in the future

```suggestion
    req.set_data(this->cameraViewController);
```

Reply:
Done here https://github.com/gazebosim/gz-gui/pull/506/commits/e59d6445eb4884b1d66663682e70d28831fff0a5